### ISSUE TITLE
Can SPCascadedropdowns be reset?

### ISSUE THREAD

**Issue by benbonus2021** [sympmarc/SPServices#86]
Once I have triggered a SPCascadedropdowns script, can I reset one of the lookup fields (remove filter, show all lookup values again)  in case that the user triggers a certain event?

If yes, how?

**Comment by sympmarc** [comment]
No, there's no method for this other than changing the value in the UI.

**Comment by benbonus2021** [comment]
Hi Marc,

thanks for your answer!

Sorry maybe that's a stupid question: What's the "UI" short term in this case?

regards

Benjamin

**Comment by sympmarc** [comment]
UI = User Interface
